server/odc-core/src/main/resources/i18n/BusinessMessages.properties

```diff
@@ -850,6 +851,11 @@ com.oceanbase.odc.notification.channel-test-message=【ODC】Channel verificatio
 com.oceanbase.odc.builtin-resource.permission-apply.project.description=Requesting [{1}] permissions for project [{0}]
 com.oceanbase.odc.builtin-resource.permission-apply.database.description=Requesting [{0}] permissions for databases
 
+#
+# Multiple Async
+#
+com.oceanbase.odc.builtin-resource.multiple-async.sub-ticket.description=[{0}]multiple database change {1} batch no.{2} {3}.{4}
```

Comment:
[{0}] Batch Database Change {1} Batch No. {2} {3}.{4}

Reply:
message recommends using lowercase in English, consistent with the rest